### ISSUE TITLE
Fail 2.5.3 Label in Name when visible label, but no accName of field is present?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#1133]
The situation is simple: a form has fields with visible labels (no `placeholder`). The programmatic connection fails (mismatch of label `for`-Attribute and `id` of fields). This would fail _1.3.1 Info and Relatonships_ - but does it also fail _2.5.3 Label in Name_ (as there is no accName at all present and labels cannot be spoken to make speech recog software focus corresponding fields)? I'd say yes. Thoughts?

**Comment by patrickhlauke** [comment]
yes, to me that's an extreme case of failure of 2.5.3 - the accname doesn't just not match/contain the visible label, it is completely absent/empty.

**Comment by alastc** [comment]
Agree with you both. Generally I try and keep failures under one SC in our (company) documentation, but that is an extreme case of 2.5.3.

Is there a doc-update that would help with this?

**Comment by patrickhlauke** [comment]
maybe a new failure technique showing this (and noting how it fails multiple SCs simultaneously)

**Comment by patrickhlauke** [comment]
(it would also fail 4.1.2 for lack of an accname, btw ... so in all, 1.3.1, 2.5.3 and 4.1.2)

**Comment by giacomo-petri** [comment]
We already have it, we just need to update the SC impacted: https://www.w3.org/WAI/WCAG22/Techniques/failures/F68

**Comment by mitchellevan** [comment]
This issue is quite similar to #2045
